src/views/v2/month/calendar-body/day/multiday-events/multiday-event/bar/inner.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * View: Month View - Single Multiday Event Inner
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/month/calendar-body/day/multiday-events/multiday-event/bar/inner.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 65a7b53a4bbb30e8a4673a8040efa229</sub>